docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -7,8 +7,8 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
-    104
+    >>> len(axl.strategies) # doctest: +SKIP
```

Comment:
Could you just remove this?

I think it's best to not be there if it's not being tested, as you say we know how many strategies there are from the diagrams.